lib/graphql-pundit/instrumenters/authorization.rb

```diff
@@ -25,15 +25,21 @@ def instrument(_type, field)
         end
 
         # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+        # rubocop:disable Metrics/CyclomaticComplexity
+        # rubocop:disable Metrics/PerceivedComplexity
+        # rubocop:disable Metrics/BlockLength
         def resolve_proc(current_user, old_resolve, options)
-          # rubocop:enable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
Why is this line removed?

Reply:
It's the last method in the file and re-enabling all disabled cops would add a lot of clutter. I'm planning on refactoring this file after this PR (or in this PR if you'd like).